src/main/java/de/dafuqs/spectrum/helpers/Support.java

```diff
@@ -26,6 +27,10 @@ public class Support {
 	public static final DecimalFormat DF = new DecimalFormat("0");
 	public static final DecimalFormat DF1 = new DecimalFormat("0.0");
 	public static final DecimalFormat DF2 = new DecimalFormat("0.00");
+
+	static {
+		DF1.setRoundingMode(RoundingMode.DOWN);
```

Comment:
I feel we should round down in all DecimalFormats to make it consistent. Pretty sure it will not be an issue anywhere.

Reply:
done.